src/content.js

```diff
@@ -3,17 +3,19 @@
 function spoilFormGet(elem) {
  // console.info({Found: elem});
 
- // Bail early if it's already one Chrome won't autodetect
- if( (!/^http/i.test(elem.getAttribute('action'))) &&
-     (!/^http/i.test(elem.action))
- ) {
-  return;
+ // Check whether the form submits to a HTTP(S) URL
+ var action = elem.action;
+ // If an input has name="action" it will mask the action property
```

Comment:
Do you have a specific failure case for the `getAttribute` check?  [This SO answer](https://stackoverflow.com/a/7590937/2877364) suggests that `getAttribute` handles the masking. 
 If we can use `getAttribute`, we won't need to touch the DOM.  However, I cheerfully yield to real testcases :) .

Reply:
The point is that the `form.action` property is always a fully resolved absolute URL (well, except when it’s the empty string, in which case we get an equally fully-resolved absolute URL from `location.href`). By contrast, `form.getAttribute('action')` will yield whatever the HTML source contains, which can be a relative URL – so we have to resolve that value against `location.href` before we can look at it.

But, hmm. Now that I write this, it occurs to me that resolving the URL isn’t actually needed. If it’s a relative URL, then resolving it against `location.href` will always produce a URL the same scheme as `location.href`. So if it’s a relative URL, we don’t care what the absolute URL is, we just look at `location.href`.

Lemme just implement that…

Reply:
Oh wow that’s 15× better than before. Thanks for sending me back to the drawing board with this one.